### ISSUE TITLE
Posts spawning on top of each other

### ISSUE THREAD

**Issue by ichbinashh** [MLG-Fortress/PopulationDensity#71]
So I'll attach some images to show you, but I'm trying to change my posts for custom ones. Upon doing so, I seem to have found an error which is causing the posts to respawn on top of the previous every time the chunk is unloaded and loaded back up again.

Happens on both Paper 20 and 32 (Latest)

https://gyazo.com/7e24734a0765c7a9561c52d0dd17ba2e
https://gyazo.com/a2a9d671ceee9e7b5456d9940accf9eb

**Comment by Apecengo** [comment]
That happens because the system detects the old post base as solid and
tries to put it on top of it.  In order to prevent that behavior it would
have to save the old post materials somehow.

**Comment by ichbinashh** [comment]
If I were to disable PopDens, remove all the previous posts and then reload it with the current block config, would this solve the issue?

**Comment by Apecengo** [comment]
It should work if you manage to remove all the posts and then changing the
materials before starting the server

**Comment by ichbinashh** [comment]
Is there a way to automatically remove posts? I'm on a test server right now, but it'll be loading on to a fresh server in the end so I don't see that it should be an issue as long as I change it before it spawns them

**Comment by Apecengo** [comment]
I see there’s a branch called remove posts, but seems outdated. I’m not
sure if it works.

Another option would be to try and set the materials for the post as AIR,
but that would leave the base “hole” in the ground (if it even works)

**Comment by ichbinashh** [comment]
I'll give that a go and report back

**Comment by ichbinashh** [comment]
Air does in fact work, but I just decided to start a fresh world and test that way. It's still duplicating them even though the world has only been loaded with custom blocks, there were no pillars originally. 

I tried using  KeepAllRegionPostsLoaded: true but that didn't seem to work either. The only post that doesn't seem to be affected is the one where my spawn is

**Comment by RoboMWM** [comment]
please give output of /version, not just arbitrary build numbers (especially since you just referenced two unrelated github issues doing that)

**Comment by ichbinashh** [comment]
My apologies, I didn't realise. I'm new to using GitHub :)

Paper(32) 1.14.1-R0.1-SNAPSHOT
Population Density Version: 5.11.2

**Comment by RoboMWM** [comment]
Your config may be useful here.

**Comment by ichbinashh** [comment]
This issue has actually been resolved for me. I'm unsure, but had a thought that it may have been due to removing the very top sign in the config. It may have also just been a 1.14.1 issue.

**Comment by RoboMWM** [comment]
Yea, it does look for a sign to locate the existing post, so that might've been the case.

**Comment by Spoonerj1** [comment]
Is it possible to remove the sign then? 
I also have recently tried disabling that top sign because it sorta looks bad (only readable from one side) and now I have region posts spawning on top of each other, creating very large towers of the base materials. I'll attach a screenshot.

version: 
<img width="644" alt="Screen Shot 2020-01-10 at 2 17 23 PM" src="https://user-images.githubusercontent.com/44783679/72187084-f45b2d00-33b3-11ea-8725-449f6283b938.png">

![2020-01-10_14 15 22](https://user-images.githubusercontent.com/44783679/72186965-b4944580-33b3-11ea-9ddb-b7dda1d77fbb.png)
![2020-01-10_14 15 18](https://user-images.githubusercontent.com/44783679/72186966-b4944580-33b3-11ea-8793-bb3a1bade5ef.png)

**Comment by RoboMWM** [comment]
> I also have recently tried disabling that top sign because it sorta looks bad (only readable from one side) 

subjective I guess. How are you removing it?

**Comment by Spoonerj1** [comment]
I used the config options to make that sign blank, the plugin does not add blank signs

**Comment by RoboMWM** [comment]
How are you doing it in the config?

**Comment by Spoonerj1** [comment]
![Screenshot_20200112-174044_Chrome](https://user-images.githubusercontent.com/44783679/72228302-b4867800-3562-11ea-9273-6bdb9a83b4dd.jpg)

**Comment by RoboMWM** [comment]
Interesting, I wasn't aware that it ignored totally blank signs. Unless @ichbinashh can confirm this is what he was doing here, probably better to make a new issue to track this.
